frontend/src/concepts/pipelines/topology/usePipelineTaskTopology.ts

```diff
@@ -19,7 +19,6 @@ export const usePipelineTaskTopology = (
 ): KubeFlowTaskTopology => {
   const tasks: PipelineRunTask[] | undefined = pipelineRun?.spec.pipelineSpec?.tasks;
   const status = pipelineRun?.status;
-
```

Comment:
These whitespace removals aren't necessary :)

Reply:
:) hehe , I dont know how I reached that file